db/db.test.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+describe('DB Test', () => {
+  beforeAll((done) => {
+    mongoose.connect('mongodb://localhost/data');
+
+    let db = mongoose.connection;
+
+    db.on('error', (err) => {
+      done.fail(err);
+    });
+
+    db.once('open', () => {
+      done();
+    });
+  });
+
+  it('If it gets here, it proves that db is connected', () => {
+    expect(1).toBe(1);
+  })
+});
```

Comment:
nice setting up the db test so far, would be a good idea to just test an insertion of a primitive data type and see if you can query for it next